components/Map/CurrentOutivityMarker.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { useMapEvents } from "react-leaflet/hooks";
+import { Marker, Popup } from "react-leaflet";
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function CurrentOutivityMarker({ outivity, position }) {
```

Comment:
hier müsst ihr die handler-function noch mitgeben, da wir nicht die state-setter-function direkt mitgeben dürfen ...
```suggestion
export default function CurrentOutivityMarker({ outivity, position, onSetUserPosition }) {
```